### PR TITLE
install all program bundles at all sites

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

**Comment by mogoodrich** [comment]
@lnball  @brandones  thoughts on this?  This is just continuing to move toward a model where we have a larger set of core metadata installed in all PIH EMR implementations.  In this case, we are basically installing all programs in all implementations.

This came up after a discussion with Vamba where he requested just being able to turn a program on/off via the component (and not have to remember to install the Metadata as well), but it was also something I had been thinking about earlier.

Interested in your thoughts... if it's a go, I will probably wait until next merge to merge so I can test to make sure it works properly for each country.

**Comment by lnball** [comment]
We still use components to have the programs "appear".  When @mogoodrich writes "install", it makes me think that they will all appear on every site and dashboard.

**Comment by mogoodrich** [comment]
@lnball No, the component will still determine whether a program appears or not... the difference is the underlying program metadata will be installed behind-the-scenes everywhere.

**Comment by mogoodrich** [comment]
(The problem previously was that, say you enable "xyzProgramComponent" but forgot to add the "xyzzProgramMetadataBundle" to your "CountryMetadataBundle" things would crash)

**Comment by lnball** [comment]
Then it's a win for me.  Less chance to forget a step.

**Comment by mogoodrich** [comment]
yeah, exactly @lnball

File: api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/mexico/MexicoMetadataToInstallAfterConceptsBundle.java
Comment (brandones) [inline api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/mexico/MexicoMetadataToInstallAfterConceptsBundle.java:25]: I think you need to add these to the PIH Programs bundle. As well as the Covid19 program.

**Comment by brandones** [comment]
Yeah, definitely an improvement. Best would be moving all this to the Iniz programs loader. Then it's also easy for sites to override programs, e.g. to change the workflows or states.

**Comment by mogoodrich** [comment]
I double checked and I believe they are all in the PIHProgramsBundle @brandones 

https://github.com/PIH/openmrs-module-pihcore/blob/master/api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/program/PihProgramsBundle.java

(Though I suspect they weren't at one point which is why you added them here)

And, yeah, Iniz is still the long-term plan.

**Comment by brandones** [comment]
Aha! Great. Thanks.

**Review by brandones — APPROVED** [review]

**Comment by mogoodrich** [comment]
Thanks @brandones  @lnball , merging this in.

Technically, we should try re-installing a SDK with each country from scratch, but I'm going to forgo that for now.

**Pull request merged**
